src/scenes/PostGig.jsx

```
@@ -287,3 +147,155 @@ export const PostGig = () => {
+      >
+        {financials.net > 0 ? '+' : ''}
+        {financials.net}€
+      </div>
+    </div>
+
+    <div className='text-center'>
+      <button
+        onClick={onNext}
+        className='bg-(--toxic-green) text-(--void-black) px-8 py-3 font-bold hover:bg-(--star-white) transition-colors uppercase'
+      >
+        Continue to Socials &gt;
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)